mypy/main.py

```diff
@@ -321,13 +321,13 @@ def add_invertible_flag(flag: str,
         ", ".join(strict_flag_names))
     parser.add_argument('--strict', action='store_true', dest='special-opts:strict',
                         help=strict_help)
+    parser.add_argument('--shadow-file', nargs=2, metavar=('SOURCE_FILE', 'SHADOW_FILE'),
+                        dest='shadow_file',
+                        help='Typecheck SHADOW_FILE in place of SOURCE_FILE. '
+                        'Primarily intended for tooling. Allows tooling to make transformations '
+                        'to a file before type checking without having to change the file '
+                        'in-place.')
```

Comment:
I would prefer a shorter (one-line) help text here and the longer explanation in the docs.